web/api/v0/__init__.py

```diff
@@ -940,3 +941,33 @@ def patch(self, token: str, password: str) -> ResponseReturnValue:
 
 
 api.add_resource(ResetPasswordResource, '/user/reset-password')
+
+
+class IdentifyResource(Resource):
+    @use_kwargs(
+        {
+            "name": fields.Str(required=True),
+            "byear": fields.Int(required=False, missing=None),
+            "uid": fields.Int(required=True),
+        },
+        location="form",
+    )
+    def post(self, name: str, byear: int, uid: int) -> ResponseReturnValue:
```

Comment:
I would suggest an `get` endpoint for that, as nothing is created/changed by it.